bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-control-plane.yaml

```diff
@@ -0,0 +1,268 @@
+            EOF
+        image: "{{.OvnImage}}"
+      containers:
+      # token-minter creates a token with the default service account path
+      # The token is read by ovn-k containers to authenticate against the hosted cluster api server
+      - name: token-minter
+        image: "{{.TokenMinterImage}}"
+        command: ["/usr/bin/control-plane-operator", "token-minter"]
+        args:
+        - --service-account-namespace=openshift-ovn-kubernetes
+        - --service-account-name=ovn-kubernetes-controller
+        - --token-audience={{.TokenAudience}}
+        - --token-file=/var/run/secrets/hosted_cluster/token
+        - --kubeconfig=/etc/kubernetes/kubeconfig
+        resources:
+          requests:
+            cpu: 10m
+            memory: 30Mi
+        volumeMounts:
+        - mountPath: /etc/kubernetes
+          name: admin-kubeconfig
+        - mountPath: /var/run/secrets/hosted_cluster
+          name: hosted-cluster-api-access
+      # ovnkube master: convert kubernetes objects in to nbdb logical network components
+      - name: ovnkube-control-plane
+        image: "{{.OvnImage}}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -xe
+          if [[ -f "/env/_master" ]]; then
+            set -o allexport
+            source "/env/_master"
+            set +o allexport
+          fi
+
+          # TLS for ovnkube-master metrics
+          TLS_PK=/etc/pki/tls/metrics-cert/tls.key
+          TLS_CERT=/etc/pki/tls/metrics-cert/tls.crt
+
+          if [ "{{.OVN_GATEWAY_MODE}}" == "shared" ]; then
+            gateway_mode_flags="--gateway-mode shared --gateway-interface br-ex"
+          elif [ "{{.OVN_GATEWAY_MODE}}" == "local" ]; then
+            gateway_mode_flags="--gateway-mode local --gateway-interface br-ex"
+          else
+            echo "Invalid OVN_GATEWAY_MODE: \"{{.OVN_GATEWAY_MODE}}\". Must be \"local\" or \"shared\"."
+            exit 1
+          fi
+
+          retries=0
+          while [ ! -f /var/run/secrets/hosted_cluster/token ]; do
+            (( retries += 1 ))
+            sleep 1
+            if [[ "${retries}" -gt 30 ]]; then
+              echo "$(date -Iseconds) - Hosted cluster token not found"
+                exit 1
+            fi
+          done
+
+          multi_network_enabled_flag=
+          if [[ "{{.OVN_MULTI_NETWORK_ENABLE}}" == "true" ]]; then
+            multi_network_enabled_flag="--enable-multi-network"
+          fi
+
+          echo "I$(date "+%m%d %H:%M:%S.%N") - ovnkube-control-plane - start ovnkube --init-cluster-manager ${K8S_NODE}"
+          exec /usr/bin/ovnkube \
+            --enable-interconnect \
+            --init-cluster-manager "${K8S_NODE}" \
+            --config-file=/run/ovnkube-config/ovnkube.conf \
+            --k8s-token-file=/var/run/secrets/hosted_cluster/token \
+            --ovn-empty-lb-events \
+            --loglevel "${OVN_KUBE_LOG_LEVEL}" \
+            --metrics-bind-address "127.0.0.1:29104" \
```

Comment:
somehow for managed we're not going through a kube-rbac-proxy container to expose metrics... I kept what we have in master and if necessary we can revisit this later on.

The discrepancy you spotted is a mistake in master. Basically in master ovnkube-master exposes its metrics on port 9102, but its pod declares port 29102. It works because the pod is host-networked and doesn't really need to define any ports, as far as I know. I fixed that in the managed/single-zone folder in this PR.